core/src/actors/chain_manager/mod.rs

```diff
@@ -225,17 +227,29 @@ impl ChainManager {
     fn build_block(&self, msg: &BuildBlock) -> Block {
         // Get all the unspent transactions and calculate the sum of their fees
         let mut transaction_fees = 0;
-        let transactions: Vec<Transaction> = self
-            .transactions_pool
-            .iter()
-            .map(|t| {
-                // TODO: t.fee()
-                // TODO: get the transactions that fit in block (https://github.com/witnet/witnet-rust/issues/262)
-
-                transaction_fees += 1;
-                t.clone()
-            })
-            .collect();
+
+        let max_block_weight = self.max_block_weight;
+        let mut block_weight = 0;
+        let mut transactions = Vec::new();
+
+        for transaction in self.transactions_pool.iter() {
+            // Currently, 1 weight unit is equivalent to 1 byte
+            let transaction_weight = transaction.size();
+            // TODO: Calculate transaction fee
+            let transaction_fee = 1;
```

Comment:
Let's take this chance to create `transaction.fee()` returning a static `1`. Then incorporating the actual fee calculation logic will be plug-and-play.